tests/unit/datastore/test_datastore.py

```diff
@@ -32,13 +32,18 @@ class TestRecord(DatastoreRecord):
             decode=lambda val: datetime.fromisoformat(val.decode()))
 
 
-def test_datastore_describe():
+def test_datastore_create():
     temp_path = tempfile.mkdtemp()
     storage = datastore.Datastore(temp_path)
     assert storage.LMDB_MAP_SIZE == 1_000_000_000_000
     assert storage.db_path == temp_path
     assert storage._Datastore__db_env.path() == temp_path
 
+
+def test_datastore_describe(mock_or_real_datastore):
```

Comment:
What's the rationale for running a datastore unit test over a mocked datastore?

Reply:
To verify that all our assumptions about how LMDB work are valid for the mock as well. This is, in fact, how I debugged the mock in the first place.

Reply:
Ah I see...makes sense!

Reply:
> To verify that all our assumptions about how LMDB works are valid for the mock as well.

A good principal for unit tests, generally.